### DISCUSSION TITLE
Bulk get user metadata?

### DISCUSSION THREAD

**Discussion by m1212e** [zitadel/zitadel#6257] [Q&A] [answered]
Is there an ednpoint to retrieve all user metadata?
Basically this as GET: https://zitadel.com/docs/apis/resources/mgmt/management-service-bulk-remove-user-metadata

**Comment by hifabienne** [comment]
hei @m1212e 
At the moment we do not have an endpoint to get all user metadata, it is currently only on a specific user.

**Reply by m1212e** [reply]
Ok, but also no endpoint to get all metadata per user? Just like the DELETE endpoint I mentioned, but for GET?

**Reply by hifabienne** [reply, marked as answer]
Searching metadata for a specific user exists, you can find the docs here: https://zitadel.com/docs/apis/resources/mgmt/management-service-list-user-metadata

**Reply by m1212e** [reply]
Ahh I see, including no queries just returns all metadata. Thanks!
